notebooks/DL1/calibration.ipynb

```diff
@@ -32,11 +32,11 @@
     "\n",
```

Comment:
<p>What is the purpose of this function? Wouldn't it just work to pass max_events to it, and if it's none, it will work fine (no need for the if-statement, I think)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/protopipe/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>